mne/decoding/receptive_field.py

```diff
@@ -47,13 +48,18 @@ class ReceptiveField(BaseEstimator):
         Defines how predictions will be scored. Currently must be one of
```

Comment:
@choldgraf Do you agree if I edit the receptive field class description slightly to also mention decoding ?
> This allows you to fit an encoding model (stimulus to brain) or a decoding model (brain to stimulus) using time-lagged input features (for example, a spectro- or spatio-temporal receptive field (STRF))

Reply:
I think is is great! (though maybe at the end maybe juts say `, or STRF` instead of having double-parentheses). We should make it more clear that it's possible to use this for continuous decoding.